SmartDeviceLink-iOS/SmartDeviceLink/SDLRPCRequestFactory.h

```diff
@@ -93,6 +94,7 @@
 + (SDLAlert *)buildAlertWithAlertText1:(NSString *)alertText1 alertText2:(NSString *)alertText2 duration:(NSNumber *)duration correlationID:(NSNumber *)correlationID;
 //*****
 
++(SDLAlertManeuver*) buildAlertManeuverwithTTSchunks:(NSMutableArray *)ttsChunks softButtons:(NSMutableArray*) softButtons correlationID:(NSNumber*) correlationID;
```

Comment:
Yeah, I did a clang-format of the whole project here: https://github.com/smartdevicelink/sdl_ios/commit/93c74ebd416aa7119d19f620d95e5767713ba4b7

That took care of all misformatting in the project